src/test/kotlin/org/jetbrains/research/transformations/PythonTransformationSetupTest.kt

```diff
@@ -3,16 +3,19 @@ package org.jetbrains.research.transformations
 import PythonTransformationsTest
 import com.github.gumtreediff.gen.python.PythonTreeGenerator
 import com.github.gumtreediff.tree.TreeContext
+import org.jetbrains.research.transformations.util.Util.getContentFromFile
+import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
+import java.io.File
 
 /*
-* Just check if the parsers work correct
+* Just check if the parsers work correctly
 * */
 internal class PythonTransformationSetupTest: PythonTransformationsTest() {
+    private val SRC_FILE: String = javaClass.getResource("source.py").path
```

Comment:
If it's not a const, don't use CAPS, see naming convention https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html